### PR TITLE
Adding OperationalInsights SDK v1.0.0 and tests

### PR DESCRIPTION
**Pull request by pilor**
This is the initial check-in for the Operational Insights .Net SDK (v1.0.0). It contains all of Operational Insight's current management operations.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@pilor__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (cheggert). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by ogail** [comment]
@azuresdkci add to whitelist

File: src/ResourceManagement/OperationalInsights/OperationalInsightsManagement/packages.config [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Hyak.Common" version="1.0.2" targetFramework="portable-net403+win+wpa81" />
+  <package id="Microsoft.Azure.Common" version="2.0.4" targetFramework="portable-net45+sl50+win+wp80" />
```
Comment (yugangw-msft) [inline src/ResourceManagement/OperationalInsights/OperationalInsightsManagement/packages.config]: please use 2.0.10

File: src/ResourceManagement/OperationalInsights/OperationalInsightsManagement/Microsoft.Azure.Management.OperationalInsights.nuget.proj [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <!--
+    Microsoft.Azure.Management.OperationalInsights
+    -->
+    <SdkNuGetPackage Include="Microsoft.Azure.Management.OperationalInsights">
+      <PackageVersion>1.0.0</PackageVersion>
```
Comment (yugangw-msft) [inline src/ResourceManagement/OperationalInsights/OperationalInsightsManagement/Microsoft.Azure.Management.OperationalInsights.nuget.proj]: usually new package starts from 0.9.0

**Comment by pilor** [comment]
Pushed changes to address yugang's comments

**Pull request merged**